src/appellate/appellate.js

```diff
@@ -200,3 +206,179 @@ AppellateDelegate.prototype.handleAttachmentPage = async function () {
     (ok) => callback(ok)
   );
 };
+
+// If this page offers a single document, intercept navigation to the document view page.
+AppellateDelegate.prototype.handleSingleDocumentPageView = async function () {
+  overwriteFormSubmitMethod();
+
+  this.pacer_case_id = await APPELLATE.getCaseId(this.tabId, this.queryParameters, this.docId);
+  await saveCaseIdinTabStorage({ tabId: this.tabId }, this.pacer_case_id);
+
+  // When we receive the message from the above submit method, submit the form
+  // via XHR so we can get the document before the browser does.
+  window.addEventListener('message', this.onDocumentViewSubmit.bind(this), false);
+
+  let callback = $.proxy(function (api_results) {
+    console.info(`RECAP: Got results from API. Running callback on API results to ` + `insert link`);
+    let result = api_results.results.filter((obj) => obj.pacer_doc_id == this.docId, this)[0];
+
+    if (!result) {
+      return;
+    }
+
+    let href = `https://www.courtlistener.com/${result.filepath_local}`;
```

Comment:
This is a link to a PDF, right? So:

```suggestion
    let href = `https://storage.courtlistener.com/${result.filepath_local}`;
```

Reply:
You're right. Thanks!